### ISSUE TITLE
Request: searching for decks of certain spheres

### ISSUE THREAD

**Issue by M-Kristof** [Sydtrack/ringsdb#47]
It would be very useful to be able to search for decks only containing cards of certain spheres. For example:  Leadership + Spirit, monosphere Lore, etc.

**Comment by Sydtrack** [comment]
Should it filter heroes only or any card?
There are some cards like Fili + Kili that are used even if there is no sphere match.

**Comment by M-Kristof** [comment]
Indeed, a filter for the spheres of the heroes makes more sense than for all the cards.
